orttraining/orttraining/core/optimizer/compute_optimizer/padding_elimination.cc

```diff
@@ -285,6 +285,7 @@ void IterateSubgraphFromNode(Graph& graph,
       PushAllOutputNode(graph, to_visit, cur, visited);
     } else if (graph_utils::IsSupportedOptypeVersionAndDomain(*cur, "Cast", {9, 13}) ||
                graph_utils::IsSupportedOptypeVersionAndDomain(*cur, "FastGelu", {1}, kMSDomain) ||
+               graph_utils::IsSupportedOptypeVersionAndDomain(*cur, "Gelu", {1, 20}) ||
```

Comment:
This is the list of valid ONNX opset versions for the operator, which is only 20 for Gelu. 

```suggestion
               graph_utils::IsSupportedOptypeVersionAndDomain(*cur, "Gelu", {20}) ||
```